src-input/duk_bi_object.c

```diff
@@ -302,8 +294,7 @@ DUK_INTERNAL duk_ret_t duk_bi_object_constructor_is_sealed_frozen_shared(duk_con
 		 * is considered to be already sealed and frozen.
 		 */
 		h = duk_get_hobject(ctx, 0);
-		duk_push_boolean(ctx, (h == NULL) ?
-		                      1 :
+		duk_push_boolean(ctx, (h == NULL) ||
```

Comment:
I had thought about this--I do stuff like that in minisphere all the time--but wasn't sure how you felt about leveraging short-circuiting like that (from a code readability standpoint).